wire_response.go

```diff
@@ -210,6 +211,15 @@ func (ii IDSet) String() string {
 	return sb.String()
 }
 
+// SortedInt64Slice returns the values in a IDSet field in a int64 slice that is
+// sorted
+func (ii IDSet) SortedInt64Slice() []int64 {
+	var idSetSlice = make([]int64, len(ii))
+	copy(idSetSlice, ii)
+	sort.Slice(idSetSlice, func(i, j int) bool { return idSetSlice[i] < idSetSlice[j] })
+	return idSetSlice
+}
+
```

Comment:
It was just cleaner / less convoluted to implement here. Can't remember the issue exactly but we have IDSet and StringSet types which are aliases for []string and []int64 or something. So I was checking type and converting things, etc. What is the down side to having this in the pilosa package? I mean, no ones asked for it yet, but sorting the output of a set fields could be a feature at some point.

But wrt to the copy, I think my thought was to prevent modifying the object itself just to be safe. But now that I think about it again, I'm not entirely sure what it's being safe from.

If you don't want this in the pilosa package, I can remove it and implement in the test package. Otherwise, I'll remove the copy and keep as is. Please let me know.

Reply:
Yeah, okay.. this was just me being dumb. Moved this logic to the test.